ECHO-Sunrise-Explore.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
Am I correct in assessing that this is a recordkeeping/archive notebook rather than one planned to be in use? Would be helpful to have a note in the Readme guiding users re what the different files in this notebook are/are for

Reply:
I have removed it from the commit. It is in another branch and got carried over here carelessly. It was a draft.